etc/scripts/docenizer.py

```diff
@@ -31,18 +34,31 @@
     'MOVQ',  # defined in MOVD:MOVQ
     'MOVSD'  # defined in MOVS:MOVSB:MOVSW:MOVSD:MOVSQ
 ]
+# Where to extract the asmdoc archive.
+ASMDOC_DIR = "asm-docs"
```

Comment:
This is a great idea but it should probably be its own argument. Can you make it work?

Reply:
This should be okay as of [094e](https://github.com/mattgodbolt/compiler-explorer/pull/785/commits/094edf7331b90130c042906c3014218f44e822b3).